core/templates/dev/head/pages/exploration_editor/translation_tab/add_audio_translation_modal_directive.html

```diff
@@ -13,6 +13,10 @@ <h3>Add Audio Translation</h3>
                                  on-file-changed="updateUploadedFile"
                                  on-file-cleared="clearUploadedFile">
             </audio-file-uploader>
+            <div ng-if="isUpdatingAudio"
```

Comment:
I think it might be better if this message goes between the license notice and the upload button.

Reply:
I think we can leave it as-is!